src/index.js

```diff
@@ -4,6 +4,22 @@ import bindActionCreators from './utils/bindActionCreators'
 import applyMiddleware from './utils/applyMiddleware'
 import compose from './utils/compose'
 
+/*
+* create a function so that we can check if the function name has been altered by minification
+* if the function has been minified and NODE_ENV !== 'production', warn the user
+*/
+function isCrushed() {}
+
+if (isCrushed.name !== 'isCrushed' && process.env.NODE_ENV !== 'production') {
```

Comment:
The CommonJS build of Redux uses Node conventions. You can explicitly use the UMD builds in `dist` folder if you’d rather avoid them.

We just went ahead with the same pattern React is using. If you use the CommonJS build of React and don’t envify `process.env.NODE_ENV`, you are running a slow development version of React in production.

From http://facebook.github.io/react/docs/getting-started.html#using-react-from-npm:

<img width="736" alt="screen shot 2016-02-07 at 01 54 44" src="https://cloud.githubusercontent.com/assets/810438/12870460/b5273fce-cd3d-11e5-9a46-dce7888de310.png">

We just follow the same convention.